core/src/main/res/drawable/overlay_button_circle_background.xml

```diff
@@ -4,7 +4,7 @@
        android:shape="rectangle">
     <gradient
         android:type="radial"
-        android:gradientRadius="60"
+        android:gradientRadius="70%p"
```

Comment:
This file is unused and can be removed, too

Reply:
I deleted them in the last commit.